sdk/healthdataaiservices/azure-health-deidentification/samples/README.md

```diff
@@ -0,0 +1,55 @@
+# Azure Health Deidentification client library for Python
+Azure Health Deidentification is Microsoft's solution to anonymize unstructured health text.
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/healthdataaiservices/azure-health-deidentification/azure/health/deidentification)
+<!-- TODO: | [Package (PyPI)](https://pypi.org/project/azure-health-deidentification/) -->
+<!-- TODO: | [API reference documentation](https://aka.ms/azsdk-python-storage-blob-ref) -->
+| [Samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/healthdataaiservices/azure-health-deidentification/samples)
+
+
+## Getting started
+
+### Prerequisites
+* Python 3.8 or later is required to use this package. For more details, please read our page on [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy).
+* You must have an [Azure subscription](https://azure.microsoft.com/free/) and an
+**Azure Deidentification Service** to use this package.
+
+### Install the package
+Install the Azure Health Deidentification client library for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-health-deidentification
+```
+
+### Create a Deidentification Service
+If you wish to create a new storage account, you can use the
+[Azure Portal](https://docs.microsoft.com/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal).
+
+### Create the client
+In order to create a Deidentification client you must obtain the **Service URL** from your Azure Deidentification Service
+
+```python
+    endpoint = os.environ["AZURE_HEALTH_DEIDENTIFICATION_ENDPOINT"]
+    endpoint = endpoint.replace("https://", "")
+    print(endpoint)
+    # example: fuf4h4bxg5b0d0dr.api.cac001.deid.azure.com
+
+    credential = DefaultAzureCredential()
+
+    client = DeidentificationClient(endpoint, DefaultAzureCredential())
+```
+
```

Comment:
Can you add a "helloworld" example here to demostrate the realtime usecase? (sdk/healthdataaiservices/azure-health-deidentification/samples/sample_realtime_deidentification.py).